src/routes/index.svelte

```diff
@@ -37,8 +37,7 @@
   <section class="splash bg-no-repeat pl-4 md:pl-24 md:pr-24">
     <div class="mt-8 md:mt-20 md:w-1/2">
       <h1 class="text-4xl font-semibold md:text-6xl width-full height-full">Build Your Web Applications at <b>hyper</b> Speed</h1>
-      <p class="text-md mt-4 mr-8 md:mt-12 md:w-3/4">Let your development team focus on creating applications that solve the problems of the business, 
-      let hyper solve the problems of the cloud.</p>
+      <p class="text-md mt-4 mr-8 md:mt-12 md:w-3/4">hyper provides a full suite of core backend services, letting you focus on what matters to your applications. Stop the bike-shedding, and start building scalable applications today.</p>
```

Comment:
hyper provides a full suite of core backend services. Focus on what matters. Stop the bike-shedding. Start building scalable applications today.